src/pyfx/view/json_lib/__init__.py

```diff
@@ -62,14 +62,7 @@
 from .node_factory import NodeFactory
 from .object import ObjectNode
 from .array import ArrayNode
-from .primitive import StringNode, IntegerNode, NumericNode, BooleanNode, NullNode
-
-DEFAULT_NODE_IMPLS = {
-    list: ArrayNode,
-    dict: ObjectNode,
-    str: StringNode,
-    int: IntegerNode,
-    bool: BooleanNode,
-    float: NumericNode,
-    type(None): NullNode
-}
+from .primitive import StringNode, IntegerNode, NumericNode, BooleanNode, NullNode, GenericNode
+from .selector import DefaultImplementationSelector
+
+DEFAULT_NODE_IMPLS = DefaultImplementationSelector()
```

Comment:
nit: Please add a new line to the end of the file

Reply:
Done